base/tools/src/main/java/com/netscape/cmstools/kra/KRAKeyRetrieveCLI.java

```diff
@@ -101,10 +101,10 @@ public void execute(CommandLine cmd) throws Exception {
             String passphrase = cmd.getOptionValue("passphrase");
             String requestId = cmd.getOptionValue("requestID");
             String requestFile = cmd.getOptionValue("input");
-            String inputFormat = cmd.getOptionValue("input-format", "xml");
+            String inputFormat = cmd.getOptionValue("input-format", "json");
             String outputFilePath = cmd.getOptionValue("output");
             String outputDataFile = cmd.getOptionValue("output-data");
-            String outputFormat = cmd.getOptionValue("output-format", "xml");
+            String outputFormat = cmd.getOptionValue("output-format", "json");
```

Comment:
Please also update the CLI help message, i.e. drop `xml` and make `json` the default.

Reply:
Ack.